src/recognizer.cc

```diff
@@ -855,7 +855,7 @@ const char* Recognizer::PartialResult()
 
         clat = decoder_->GetLattice(decoder_->NumFramesInLattice(), false);
         if (model_->winfo_) {
-            WordAlignLatticePartial(clat, *model_->trans_model_, *model_->winfo_, 0, &aligned_lat);
```

Comment:
I'm sorry, why do you change this? It is critical for functionality. Your kaldi changes in https://github.com/kaldi-asr/kaldi/pull/4963 are also breaking things since we use different openfst version (our own branch) from Kaldi.

Reply:
Because it did not compile: Error: ‘WordAlignLatticePartial’ was not declared in this scope

So how would fix this then?

Reply:
Let me look at the test...

Reply:
We need to merge this patch into main kaldi. And probably other changes from our vosk branch